kubejs/server_scripts/base/recipes/eccentrictome/shapeless.js

```diff
@@ -63,13 +69,20 @@ ServerEvents.recipes((event) => {
                             tag: { 'patchouli:book': 'pneumaticcraft:book' }
                         }
                     },
+                    powah: {
+                        0: {
+                            Count: 1,
+                            id: 'powah:book'
+                        }
+                    },
                     rftoolsbase: {
                         0: {
                             Count: 1,
                             id: 'rftoolsbase:manual'
                         }
                     }
-                }
+                },
+                'eccentrictome:version': 1
```

Comment:
This part scares me. @MuteTiefling why'd you omit it the first time around, any particular reason?

Reply:
Wasn't intentional. It just didn't show up in the kjs_hand output.

Reply:
Older recipe worked fine ^^" It just showed up for me with kjs_hand so I added it.